test/mocks/MockTimescalesService.scala

```diff
@@ -34,8 +34,12 @@ trait MockTimescalesService extends MockFactory {
         .save(_: JsValue))
         .expects(timescales)
 
-    def get(): Future[RequestOutcome[JsValue]] =
+  def updateTimescaleTable(js: JsObject): CallHandler[Future[RequestOutcome[JsObject]]] =
       (mockTimescalesService
-        .get())
+        .updateTimescaleTable(_: JsObject))
+        .expects(js)
+    // def get(): Future[RequestOutcome[JsValue]] =
```

Comment:
As above ? Keeping for a specific reason

Reply:
removed